centaur/src/main/resources/standardTestCases/three_step.test

```diff
@@ -0,0 +1,11 @@
+name: threeStepCwl
+testFormat: workflowsuccess
+workflowType: CWL
+workflowTypeVersion: v1.0
+
+files {
+  wdl: three_step/three_step.cwl
+  inputs: three_step/inputs.json
+}
+
+metadata { }
```

Comment:
Expectations here too?

Reply:
As I understand your PR will facilitate the emitting of outputs.  When merged I will add meaningful expectations.